lib/grape/util/xml.rb

```diff
@@ -0,0 +1,13 @@
+begin
+  require 'multi_xml'
+rescue LoadError
+end
+
+module Grape
+  if Object.const_defined? :MultiXml
+    Xml = ::MultiXml
+  else
+    Xml = ::ActiveSupport::XmlMini
+    Xml::ParseError = StandardError
+  end
```

Comment:
Likewise, I'd like to declare `require 'active_support/xml_mini'` explicitly. 
At the moment we don't declare `require 'active_support'` itself.